client/src/main/java/io/confluent/kafka/schemaregistry/client/CachedSchemaRegistryClient.java

```diff
@@ -88,58 +92,55 @@ public CachedSchemaRegistryClient(
   public CachedSchemaRegistryClient(
       List<String> baseUrls,
       int identityMapCapacity,
-      SchemaProvider provider,
+      List<SchemaProvider> providers,
       Map<String, ?> originals) {
-    this(new RestService(baseUrls), identityMapCapacity, provider, originals, null);
+    this(new RestService(baseUrls), identityMapCapacity, providers, originals, null);
   }
 
   public CachedSchemaRegistryClient(
       RestService restService,
       int identityMapCapacity,
       Map<String, ?> configs) {
-    this(restService, identityMapCapacity, new AvroSchemaProvider(), configs, null);
+    this(restService, identityMapCapacity, null, configs, null);
```

Comment:
why null instead of an empty list?

Reply:
A later constructor will check if either null or empty list is passed, and if so, add the default providers.